Models/SessionCandidateModels.ts

```diff
@@ -0,0 +1,16 @@
+export interface CandidateTestStatus {
+    testName: string;
+    testStatus: string;
+}
+export interface SessionCandidate {
+    firstName: string;
+    lastName: string;
+    testStatuses: CandidateTestStatus[];
+}
+export interface NewTestSubmission{
+    testId: number;
+    testAnswer: string;
+}
+export interface TestLibraryStepperProps {
+    candidateTestStatuses: CandidateTestStatus[];
+}
```

Comment:
Props should still probably live with their component.
(because they only make any sense in that file)